kognic-io/kognic/io/tools/scene_input.py

```diff
@@ -10,19 +9,21 @@
     AggregatedLidarsAndCamerasSequence, InputStatus
 from kognic.io.model.input.feature_flags import FeatureFlags
 from kognic.io.resources.abstract import CreateableIOResource
-from kognic.io.tools.model import SceneInput, CreationResult, CreationError, CreationStage, CreationStatus, Scene, SceneUuid
+from kognic.io.tools.model import SceneWithPreannotation, CreationResult, CreationError, CreationStage, CreationStatus, Scene, SceneUuid
 
 logger = logging.getLogger(__name__)
 
+NORMAL_CHUNK_SIZE = 1000
```

Comment:
I don't think that 1000 is ever going to work actually. Maybe 100 is more reasonable
```suggestion
NORMAL_CHUNK_SIZE = 100
```

Reply:
Oops, used it for testing, forgot to change back.